pacemaker_changenotifier/lib/models/workout_model.dart

```diff
@@ -1,17 +1,6 @@
 import 'package:uuid/uuid.dart';
 
 class Workout {
-  final String id;
-  final String workout;
-  final String week;
-  final String weekday;
-  final String km;
-  final String time;
-  final String pace;
-  final String intensity;
-  final String heartrate;
-  final bool complete;
-
   Workout({
     String id,
```

Comment:
if you have parameters, that should be always not null, you can use @required annotation